### PR TITLE
@craigspaeth: calendar event urls

### PR DESCRIPTION
**Pull request by xtina-starr**
Since this will be reused by mg, I added some calendar integration to the mixins https://github.com/artsy/force/issues/3368

### CODE REVIEW

File: lib/calendar_urls.coffee [hunk 1]
```diff
@@ -0,0 +1,83 @@
+ADDRESS_ATTR = ''
+TITLE_ATTR = ''
+
+module.exports = ({ address_attr, title_attr }) ->
```
Comment (craigspaeth) [inline lib/calendar_urls.coffee]: camelCase plz! `addressAttr` or even just `address`

Comment (broskoski) [inline lib/calendar_urls.coffee]: Agh, sorry @xtina-starr that was my fault!

**Comment by craigspaeth** [comment]
:clap: well done! Minor comment above, but in the interest of solving this upstream before it affects downstream users—I'm afraid I'm going to have to hold up the PR for it. Afterwards feel free to merge!

File: lib/calendar_urls.coffee [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  ADDRESS_ATTR = address_attr
+  TITLE_ATTR = title_attr
+  module.exports.methods
+
+module.exports.methods =
+  formatTime: formatTime = (date) ->
+    new Date(date).toISOString().replace(/-|:|\.\d+/g, '')
+
+  calulateDurationTime: calulateDurationTime = (end, start) ->
```
Comment (craigspaeth) [inline lib/calendar_urls.coffee]: Neat way to hoist a local variable here! `calulateDurationTime: calulateDurationTime =`

File: lib/calendar_urls.coffee (hunk 1 above)
Comment (craigspaeth) [inline lib/calendar_urls.coffee]: For repenting your snake_case ways, your syntax sins shall be forgiven Cab :pray:

**Pull request merged**